### DISCUSSION TITLE
Commit-build icon's popup not displaying correctly

### DISCUSSION THREAD

**Discussion by alexander-me** [community/community#38489] [Pull Requests]
![Screenshot 2022-11-07 174254](https://user-images.githubusercontent.com/117652843/200352376-a83d2764-08d5-461b-bb54-6a85f9538c22.png)

Some information in popup is cut off.

This issue happens when footer does not load.

**Comment by david-engelmann** [comment]
@alexander-me What browser and is there extensions installed?

**Reply by alexander-me** [reply]
> @alexander-me What browser and is there extensions installed?

Browser: Google Chrome 107.0.5304.88
I tried to reproduce this bug again after a while, but I couldn't.
As far as I understand - this happened when my laptop was overloaded and the browser did not render the site footer.

I think we can close this issue.
Sorry for disturbing.
